daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -31,7 +31,7 @@ import scala.util.control.NoStackTrace
 
 private[lf] object Speedy {
 
-  // Would like these to have zero cost when not enabled. Better still, to be switchable at runtime.
+  // These have zero cost when not enabled. But they are not switchable at runtime.
```

Comment:
Boy scouting: as we are in an object, can't we replace `private[this]` by `private`?

Reply:
yeah, probably